arbitrum-docs/arbos/l1-to-l2-messaging.mdx

```diff
@@ -1,3 +1,32 @@
+import { BigNumber } from "@ethersproject/bignumber";
+import { useEffect, useState } from "react";
+
+export const AddressAliasHelper = () => {
```

Comment:
Can we keep all the react components in src/components and import them into the mdx files? For cleanliness and b/c we may went to reuse them :)